kerberos/tests/integration_test.rs

```diff
@@ -0,0 +1,32 @@
+use kerberos::{
```

Comment:
Refactor needed:
1. Create folder components/ under tests/
2. Create main.rs under components/
3. Add mod client_server; to mains.rs
4. Create a file client_server.rs and then move these codes to that file.
5. Remove integration_tests.rs